local/cleanup.sh

```diff
@@ -22,6 +22,7 @@
 
 docker compose down -v
 docker compose -f docker-compose-infrastructure.yaml down -v
+docker compose -f docker-compose-idp.yaml down -v
```

Comment:
Does this file exist in this project?

Reply:
Great catch. Missed to remove that. Used initially a separate docker compose file to easier spin up the idp only during testing and configuration of the realm.

Removed / Fixed :)